x-pack/plugins/fleet/common/authz.test.ts

```diff
@@ -10,8 +10,11 @@ import { DEFAULT_APP_CATEGORIES } from '@kbn/core-application-common';
 import { TRANSFORM_PLUGIN_ID } from './constants/plugin';
 
 import {
+  calculateEndpointExceptionsPrivilegesFromCapabilities,
```

Comment:
I forgot to ask this before: but were fleet changes really necessary for the endpoint exceptions authz?

We normally only have to touch fleet authz for when one of our kibana privileges needs to be used in fleet to allow access to one of the APIs. (right?)

Reply:
I believe it was necessary to add this to fleet authz same as we did for endpoint package privileges, as privilege verification is done via fleetAuthz for all endpoint/exception_list artifact APIs.

Reply:
I don't know about that. We only run our privileges through fleet so that we can enable fleet APIs for kibana features controls that are applicable to endpoint management. 

For endpoint exceptions, I don't think they need access to fleet APIs (ex. they don't support the "per policy" functionality), so it should all have been done from within Security Solution? (ex. that's how it works today, prior to RBAC being introduced for serverless).